contribute/guide.md

```diff
@@ -57,100 +48,37 @@ If you are interested in contributing code, we ask you to sign the
 
-<br/>
-
-### I have this idea that I'd like to add to Scala, how do I start?
-
-The first step to making a change is to discuss it with the community at large, to make sure everyone agrees on the idea
-and on the implementation plan. Starting point ("community") bugs are usually uncontroversial, so you can jump right
-ahead to hacking the scala source tree and filing a pull request. For larger changes it is best to announce the change
-on the [scala-internals](http://groups.google.com/group/scala-internals) mailing list and get developer feedback. For really complex changes, a [Scala Improvement Process (SIP)](http://docs.scala-lang.org/sips/) document might be required, but the first step is always to discuss it on the mailing list.
-
-Contributions, big or small, simple or complex, controversial or undisputed, need to materialize as patches against
-the Scala project source tree. The [hacker guide](hacker-guide.html) will explain how to materialize your idea into a full-fledged pull request against the Scala code base.
-
-<!--
-
-To be moved to hacker-guide.md:
-
-- Subpages or subsections for contributing for each project (i.e. how to find tickets/features to work on)
-- Building and contributing code (contributor's workflow)
-  - link to a proper git etiquette page
-  - how to run and use partest
-  - compiler page (links to compiler-related docs i.e. reflection/macros/compiler-plugin guides, links to compiler internal videos, and useful info from wiki) (or should this go on docs.scala-lang?)
-- Link to how to make a bug report
-
-The source code is hosted on [github](http://github.com/scala/scala).
-
-If you're interested in joining our community and contributing to the project,
-start simple- often Scaladoc (Scala's javadoc-like tool, Example: [Scala
-Standard Library API](www.scala-lang.org/api/current/index.html#package)) is
-the best place to get started.
-
-
-## Overview of the Scala Ecosystem
-
-[Scala Project](http://scala-lang.org)
-
-[Typesafe Stack](http://typesafe.com/stack)
-
-        EPFL             Typesafe Stack
-     _______________      ___________
-    |               |    |           |
-    | stdlib        |    |  Akka     |
-    | compiler      |    |  Play!    |
-    | scaladoc      |    |  SBT      |
-    | documentation |    |  ....     |
-    |_______________|    |___________|
+### Bug-fix Check List
 
+This is the impatient developer's checklist for the steps to submit a bug-fix pull request to the Scala project. For more information, description and justification for the steps, follow the links in that step. Further specific instructions for the release of Scala you are targeting can be found in the `CONTRIBUTING.md` file for that [github branch](https://github.com/scala/scala)
 
-## Basic Workflow
+1. [Select a bug to fix from JIRA](https://issues.scala-lang.org/issues/?filter=12111), or if you found the bug yourself and want to fix it, [create a JIRA issue](./bug-reporting-guide.html) (but please 
```

Comment:
Minor: I recommend avoiding filters, just put the query in the URL: https://issues.scala-lang.org/issues/?jql=status%20%3D%20Open%20AND%20labels%20%3D%20community%20ORDER%20BY%20component